src/Catty/ViewController/Continue&New/SceneTableViewController.m

```diff
@@ -147,12 +114,47 @@ - (void)addObjectActionWithName:(NSString*)objectName
     ltvc.showAddLookActionSheetAtStartForScriptEditor = NO;
     ltvc.afterSafeBlock =  ^(Look* look) {
         [self.navigationController popViewControllerAnimated:YES];
+        NSString *initialText = @"";
+        if (look) {
+            if (![look.name  isEqual: @"look"]) {
```

Comment:
What is this for?
Default name for drawn images is "Look" (localized), when is this triggered?

Reply:
Apparently it used for showing the name of the downloaded image from the media library. At the moment it looks like: 1) Media Library -> the name is prefilled with the name on the media library.
2) Drawing Image -> there is no prefilled name. It's empty.
3) choosing Image from phone -> the name is prefilled with "My image"
Is it ok when the prefilled string is empty for self drawing images?